File: Python Code/Daily_Flash_Code_0/Week 8/11 Mar/44-DailyFlash_Solutions/44-DailyFlash_Solutions/11-Mar/Python/prog1.py
for num in range(1, 61):
    temp = num
    cnt = 0
    while(temp != 0):
        temp = int(temp / 10)
        cnt += 1
    sqr = 0
    temp = num * num
    while(cnt > 0):
        sqr = sqr * 10 + temp % 10
        temp = int(temp / 10)
        cnt -= 1
    rev = 0
    while(sqr != 0):
        rev = rev * 10 + sqr % 10
        sqr = int(sqr / 10)
    if(rev == num):
        print(num, end = " ")

File: Python Code/Daily_Flash_Code_0/Week 8/11 Mar/44-DailyFlash_Solutions/44-DailyFlash_Solutions/11-Mar/C/prog1.c
#include<stdio.h>

int main() {
	int num, cnt = 0, sqr = 0;
	for(int i = 1; i <= 60; i++) {
		int temp = i;
		cnt = 0;
		while(temp != 0) {
			temp /= 10;
			cnt++;
		}
		temp = i * i;
		sqr = 0;
		while(cnt != 0) {
			sqr = sqr * 10 + temp % 10;
			temp /= 10;
			cnt--;
		}
		int rev = 0;
		while(sqr != 0) {
			rev = rev * 10 + sqr % 10;
			sqr /= 10;
		}
		if(rev == i)
			printf("%d ", i);
	}
}	
